src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -179,14 +83,18 @@ def _list_devices(self, cmd):
 
             return HandleCommandResult(stdout=result)
 
-    def _list_services(self, cmd):
-        hostname = cmd.get('host', None)
-        svc_id = cmd.get('svc_id', None)
-        svc_type = cmd.get('svc_type', None)
+    @CLIReadCommand('orchestrator service ls',
+                    "name=host,type=CephString,req=false "
+                    "name=svc_type,type=CephString,req=false "
```

Comment:
Is `svc_type` not a good candidate for a `CephChoices`?

Reply:
yep.